### ISSUE TITLE
Error running kaa

### ISSUE THREAD

**Issue by alexmigf** [kaaedit/kaa#136]
I have installed Kaa successfully but now i can't run it:

```
Traceback (most recent call last):
  File "/usr/local/bin/kaa", line 9, in <module>
    load_entry_point('kaaedit==0.52.0', 'console_scripts', 'kaa')()
  File "/home/alexandre/.local/lib/python3.4/site-packages/kaa/cui/main.py", line 219, in run
    curses.wrapper(main)
  File "/usr/lib/python3.4/curses/__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
  File "/home/alexandre/.local/lib/python3.4/site-packages/kaa/cui/main.py", line 100, in main
    kaa.app = app.CuiApp(conf)
  File "/home/alexandre/.local/lib/python3.4/site-packages/kaa/cui/app.py", line 32, in __init__
    self.clipboard = clipboard.select_clipboard()
  File "/home/alexandre/.local/lib/python3.4/site-packages/kaa/clipboard.py", line 13, in select_clipboard
    elif X11Clipboard.check():
  File "/home/alexandre/.local/lib/python3.4/site-packages/kaa/clipboard.py", line 106, in check
    if not 'DISPLAY' in os.environ:
NameError: name 'os' is not defined
```

**Comment by alexmigf** [comment]
I have fix this putting:

`import os`

on the clipboard.py

**Comment by atsuoishimoto** [comment]
Thanks! I'll release new version later.

**Comment by atsuoishimoto** [comment]
fixed in 0.53
